packages/maskbook/src/plugins/Trader/apis/0x/index.ts

```diff
@@ -20,7 +33,8 @@ export async function swapQuote(request: SwapQuoteRequest) {
         params.set('buyTokenPercentageFee', new BigNumber(request.buyTokenPercentageFee).dividedBy(100).toFixed())
     if (request.includedSources) params.set('includedSources', request.includedSources.join())
     if (request.excludedSources) params.set('excludedSources', request.excludedSources.join())
-    const response = await fetch(`${ZRX_BASE_URL}/swap/v1/quote?${params.toString()}`)
+
+    const response = await fetch(`${getBaseURL(currentChainIdSettings.value)}/swap/v1/quote?${params.toString()}`)
```

Comment:
`currentNetworkTypeSettings.value`

Reply:
Don't forget to change `SwapQuoteRequest`in `packages/maskbook/src/plugins/Trader/trader/0x/useTrade.ts`.

Reply:
done